Cargo.toml

```diff
@@ -40,16 +40,14 @@ harness = false
 
 [dependencies]
 backtrace = { version = "0.3", optional = true }
-subtle = "2.3"
 crossbeam-utils = "0.8"
 ff = "0.9"
 group = "0.9"
 metrics = "0.14.2"
 num_cpus = "1.13"
+pasta_curves = { git = "https://github.com/zcash/pasta_curves.git", branch = "main" }
```

Comment:
Can this be specific to the revision currently on `main`? We don't want changes to `pasta_curves`'s git repo to break halo2 code suddenly.

Reply:
Yep, I was waiting for the other PR to merge. I'll update this after https://github.com/zcash/pasta_curves/pull/2 is merged.